schema/artwork.js

```diff
@@ -73,6 +73,24 @@ let ArtworkType = new GraphQLObjectType({
         type: Artist.type,
         resolve: ({ artist }) => gravity(`artist/${artist.id}`)
       },
+      contact_label: {
+        type: GraphQLString,
+        resolve: ({ partner }) => {
+          if (partner.type == 'Gallery') {
+            return 'Gallery';
+          } else {
+            return 'Seller';
+          }
+        }
+      },
```

Comment:
This doesn't seem *super* useful—could just be a lil helper in the template where it's relevant.